.babelrc

```diff
@@ -1,7 +1,7 @@
 {
   "presets": [
     "react",
-    ["env", {"targets": {"browsers": ["firefox >= 54"]}}]
```

Comment:
@jimporter this doesn't work presumably because nothing is JS expected to be "babel"-ed. should we just keep this at 54? looking at [plugins.json](https://github.com/babel/babel-preset-env/blob/master/data/plugins.json) maybe consider 55?